blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -251,7 +251,7 @@ export class RunDetailsPipeline extends Component {
             return <QueuedState />;
         }
         const { nodeKey, supportsNode, noSteps, currentSteps, hasResultsForSteps, isPipelineQueued } = this.mergedConfig;// supportsNodes(run);
-        const resultRun = run.isCompleted() ? run.state : run.result;
+        const resultRun = !run.isCompleted() ? run.state : run.result;
```

Comment:
Nothing really to do with this change, but this conflation of `run.state` and `run.result` really does smell to me.